src/components/Shimmer/Shimmer.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react"
+import styled, { keyframes } from "styled-components"
+
+type heightType = "small" | "medium" | "large"
+
+export interface IShimmerProps {
```

Comment:
You can `import { BaseComponentProps } from "../../common/types"` for `style` & `className`, so in future anything general like style & className need to be added to all component, can be added to `BaseComponentProps`